src/index.ts

```diff
@@ -78,15 +100,66 @@ app.get("/api/fashion-items", async (c) => {
   const items =
     conditions.length > 0
       ? await db
-          .select()
-          .from(schema.fashionItems)
-          .where(sql`${conditions.join(" AND ")}`)
-          .all()
+        .select()
+        .from(schema.fashionItems)
+        .where(
+          and(...conditions)
```

Comment:
we need to spread the AND conditions into an `and` operator instead of stringifying them. when they're stringified they show up as `object Object`